.vscode/settings.json

```diff
@@ -10,7 +10,7 @@
 		"tgui/.pnp.*": true
 	},
 	"editor.codeActionsOnSave": {
-		"source.fixAll.eslint": true
+		"source.fixAll.eslint": "explicit"
 	},
```

Comment:
this doesn't look intentional

Reply:
ughh vscode keeps doing this automatically and i have to keep uncommitting it.